src/generators/endpoint.ts

```diff
@@ -0,0 +1,181 @@
+import uuidv4 from 'uuid/v4';
+import {names} from './names';
+import {loadDoc} from '../dumper';
+import ora from 'ora';
+
+const command = 'endpoint [options]';
+
+const describe = 'generate endpoint data';
+
+const builder = {
+    endpoint_count: {
+        alias: 'n',
+        describe: 'number of endpoint to populate',
+        default: 100,
+        type: 'number',
+    },
+    doc_count: {
+        alias: 'd',
+        describe: 'number of document per endpoint',
+        default: 50,
+        type: 'number'
+    },
+};
+
+function handler(argv: any): void {
+    const spinner = ora({
+        text: 'Generating endpoint data',
+    });
+    spinner.start();
+    generateDocs(argv);
+    spinner.succeed()
+}
+
+async function generateDocs(argv: any) {
```

Comment:
don't see the need for this function to be `async` - unless something in the code below is missing `await`